source/fb_nextgen_mmm_v20.0.func.R

```diff
@@ -741,16 +747,25 @@ f.mmm <- function(...
 
   ## set paralle backend
   cat("\nRunning", iterRS,"random search trails with",lambda.n,"trails lambda cross-validation each on",set_cores,"cores...\n")
-  cl <- makeCluster(set_cores)# makeSOCKcluster(set_cores) #makeCluster(set_cores)
-  registerDoSNOW(cl)
+  #cl <- makeCluster(set_cores)# makeSOCKcluster(set_cores) #makeCluster(set_cores)
```

Comment:
This is the part we need to solve for parallel computing on nevergrad

Reply:
Yes!